bower.json

```diff
@@ -12,6 +12,7 @@
     "fontawesome": "~4.3.0",
     "highcharts": "~4.1.5",
     "jquery": "^1.11.1",
+    "jquery-mousewheel": "^3.1.12",
```

Comment:
the inclusion of jquery-mousewheel prevents scrolling outside the select menu, which is currently broken.